superset/viz.py

```diff
@@ -1300,9 +1300,9 @@ def get_data(self, df):
             for i, v in ys.iteritems():
                 x = i
                 if isinstance(x, (tuple, list)):
-                    x = ', '.join([str(s) for s in x])
+                    x = ', '.join([str(s.encode('utf8')) for s in x])
```

Comment:
I think you want to use `six.text_type(s)` instead.

Reply:
time to upgrade to py3!

Reply:
@john-bodley I tried six.text_type but it didn't work. When 'encode()' is called, by default ascii encoding scheme is used so it will break unless utf-8 is specified.

Reply:
@john-bodley I removed the str(). six.text_type makes sense now.